[2020-01-17 11:08:24,710] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: getGmailAttachments.hello_task 2020-01-17T11:08:01.837142+00:00 [queued]>
[2020-01-17 11:08:24,752] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: getGmailAttachments.hello_task 2020-01-17T11:08:01.837142+00:00 [queued]>
[2020-01-17 11:08:24,754] {{taskinstance.py:866}} INFO - 
--------------------------------------------------------------------------------
[2020-01-17 11:08:24,755] {{taskinstance.py:867}} INFO - Starting attempt 1 of 1
[2020-01-17 11:08:24,756] {{taskinstance.py:868}} INFO - 
--------------------------------------------------------------------------------
[2020-01-17 11:08:24,783] {{taskinstance.py:887}} INFO - Executing <Task(PythonOperator): hello_task> on 2020-01-17T11:08:01.837142+00:00
[2020-01-17 11:08:24,787] {{standard_task_runner.py:52}} INFO - Started process 31520 to run task
[2020-01-17 11:08:25,000] {{logging_mixin.py:112}} INFO - [2020-01-17 11:08:25,000] {{dagbag.py:403}} INFO - Filling up the DagBag from /usr/local/airflow/dags/email.py
[2020-01-17 11:08:25,017] {{logging_mixin.py:112}} INFO - Running %s on host %s <TaskInstance: getGmailAttachments.hello_task 2020-01-17T11:08:01.837142+00:00 [running]> 7f68746faaa4
[2020-01-17 11:08:25,037] {{python_operator.py:105}} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=getGmailAttachments
AIRFLOW_CTX_TASK_ID=hello_task
AIRFLOW_CTX_EXECUTION_DATE=2020-01-17T11:08:01.837142+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2020-01-17T11:08:01.837142+00:00
[2020-01-17 11:08:26,845] {{logging_mixin.py:112}} INFO - An exception occurred
[2020-01-17 11:08:26,848] {{python_operator.py:114}} INFO - Done. Returned value was: Hello world!
[2020-01-17 11:08:34,694] {{logging_mixin.py:112}} INFO - [2020-01-17 11:08:34,694] {{local_task_job.py:103}} INFO - Task exited with return code 0
